Applications/DataExplorer/mainwindow.h

```diff
@@ -47,7 +49,6 @@ class MainWindow : public QMainWindow, public Ui_MainWindowClass
 
 public:
 	MainWindow(QWidget* parent = 0);
-	~MainWindow();
```

Comment:
No, re-including the destructor didn't help.
We tested a bit on Lars' machine and the only way we could fix it was to remove the forward declarations and really include the headers to the models here. I'm currently rebasing the changes and update asap.